Add ClickPipes Protobuf documentation (#5428)
ClickHouse/clickhouse-docs: docs/integrations/data-ingestion/clickpipes/kafka/02_schema-registries.md

## How it works {#how-schema-registries-work}

ClickPipes dynamically retrieves and applies the schema from the configured schema registry.
- If there's a schema id embedded in the message, it will use that to retrieve the schema.
- If there's no schema id embedded in the message, it will use the schema id or subject name specified in the ClickPipe configuration to retrieve the schema.
- If the message is written without an embedded schema id, and no schema id or subject name is specified in the ClickPipe configuration, then the schema will not be retrieved and the message will be skipped with a `SOURCE_SCHEMA_ERROR` logged in the ClickPipes errors table.
- If the message does not conform to the schema, then the message will be skipped with a `DATA_PARSING_ERROR` logged in the ClickPipes errors table.

> - For Protobuf schemas only, ClickPipes will load any imported schemas defined as dependencies.  Avro schemas with external references are not yet supported.

Reviewer: ```suggestion
- For Protobuf schemas only, ClickPipes will load any imported schemas defined as dependencies. Avro schemas with external references aren't yet supported.
```
